### DISCUSSION TITLE
unable to get submissions beyond 20 even upon setting limit to 21

### DISCUSSION THREAD

**Discussion by pruthi41** [alfaarghya/alfa-leetcode-api#14] [Q&A] [answered]
https://alfa-leetcode-api.onrender.com/:userid/submission?limit=21

still it shows only 20 submissions, how to get beyond that

**Comment by alfaarghya** [comment, marked as answer]
LeetCode only shows the last 20 submissions, So there is no way u can fetch the last 21 submissions.
